src/components/Routes/Track/Results/ResultsChart/ResultsChart.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { ResponsivePie } from '@nivo/pie';
+import { turnDashesIntoSpacesAndCapitalize } from '../../../../../shared/utilities';
+
+const ResultsChart = (props) => {
+  const { savedApplications } = props;
+
+  const values = savedApplications.reduce((total, current) => {
+    total[current.result] = (total[current.result] || 0) + 1;
+
+    return total;
+  }, {});
+
+  const data = [];
+  for(let value in values) {
```

Comment:
`for(...in)` almost never gives you what you really want, as it iterates through every key in the object. Here's a refactor with `Object.keys`

```
const data = Object.keys(values).map((value) => {
  const name = value === '' ? 'None' : turnDashesIntoSpacesAndCapitalize(value);
  return {
    id: name,
    label: name,
    value: values[value]
  }
})
```

Reply:
[Related reading](https://eslint.org/docs/rules/guard-for-in)